static/js/property-detail/video.js

```diff
@@ -96,42 +96,18 @@ class VideoDetail {
 
     const video = document.querySelector('.media-modal-video');
 
-    if (this.dashHref) {
-      const player = window.dashjs.MediaPlayer().create();
-      player.extend('RequestModifier', () => {
-        return {
-          modifyRequestHeader: (xhr) => {
-            xhr.setRequestHeader('Authorization', `Bearer ${API.jwt}`);
-            return xhr;
-          },
-          modifyRequestURL: (url) => {
-            return url;
-          },
+    if (shaka.Player.isBrowserSupported() && (this.dashHref || this.hlsHref)) {
```

Comment:
This wasn't part of the functionality before, but it would be good to have a warning if the browser isn't supported (if there are even that many unsupported browsers)

Reply:
Essentially they only browsers that are not supported are IE <= 10 and everything on iOS (which was the case before as well).